system_setup/common/wsl_conf.py

```diff
@@ -131,6 +131,9 @@ def default_loader(is_advanced):
 class WSLConfig:
     def __init__(self, conf_file):
         self.conf_file = conf_file
+        conf_parent_folder = path.dirname(path.abspath(conf_file))
+        if not path.exists(conf_parent_folder):
+            mkdir(conf_parent_folder)
```

Comment:
You should prefer mkdirall to mkdir here, in case the parent of parent doesn’t exists.